src/Avalonia.Native/Avalonia.Native.csproj

```diff
@@ -30,4 +26,8 @@
 
   <Import Project="..\..\build\DevAnalyzers.props" />
   <Import Project="..\..\build\TrimmingEnable.props" />
+
+  <ItemGroup>
+    <Compile Remove="..\Shared\ModuleInitializer.cs" />
```

Comment:
I'm not sure that this is the correct fix. Do we really need to include this file in all projects by using `Directory.Build.props` if it's going to be visible via `InternalsVisibleTo` in `Avalonia.Base` anyway?

@kekekeks ?

Reply:
Ideally, we should consolidate way how we share internal code between assemblies.
Right now we do it in multiple ways:
- Define internal types in the Avalonia.Base and add "internal visible to" (reactive helpers, trimming attributes)
- Define shared file that needs to be included in each project where it's needed (nullable, module initializers). We also use "props" files to include them in many places at once.
- Or mix of both, internal files defined in the Avalonia.Base and then also included in some projects independently.

This PR fixes warnings caused by the mix of two of these approaches.

We discussed with @kekekeks that first option should be chosen to avoid problems like this warning. 
But alternatively, we could create a whole separated assembly just for shared internals, which will have internal visible to every other assembly. Personally, I am not a fan of yet another assembly in the folder, unless we can merge it without any trickery with cecil.